### PR TITLE
Generate API signature

### PR DESCRIPTION
**Pull request by Noctem**
Fixes the Unknown6 API checks.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,4 +8,4 @@ gpsoauth==0.3.0
 coveralls==1.1
 werkzeug==0.11.10
 sqlalchemy==1.0.14
--e git+https://github.com/tejado/pgoapi.git@0811db23d639039f968a82e06c7aa15a0a5016b6#egg=pgoapi
+-e git+https://github.com/keyphact/pgoapi.git#egg=pgoapi
```
Comment (Noctem) [inline requirements.txt]: This can return to @tejado's repo as soon as he merges in the fixes from tejado/pgoapi#182.

**Comment by adiran** [comment]
i tried this but still don't get any pokemon... is this working for you  @Noctem?

**Comment by Noctem** [comment]
@adiran It is. Do you have the encryption library? You'll have to acquire that elsewhere and set the `Encrypt:` config variable to its name (i.e. _libencrypt.so_, _libencrypt.dll_, or _libencrypt.dylib_).

**Comment by Kieken72** [comment]
@Noctem Giving me AttributeError:
`Traceback (most recent call last):
  File "worker.py", line 347, in <module>
    spawn_workers(workers, status_bar=args.status_bar)
  File "worker.py", line 281, in spawn_workers
    start_worker(worker_no, points[worker_no])
  File "worker.py", line 269, in start_worker
    points=points
  File "worker.py", line 85, in __init__
    self.api.activate_signature(config.ENCRYPT)
  File "c:\Python27\lib\site-packages\pgoapi\pgoapi.py", line 89, in __getattr__
    raise AttributeError
AttributeError`

**Comment by thiemok** [comment]
@Kieken72 did you change your own config file? Is the spelling correct?

**Comment by puddingbrood** [comment]
Did you adjust all 3 of the files?
Do you have the encryption files in the same folder?
Did you name the correct encryption files in the config? (encrypt.dll for windows)
Did you run `pip install -r requirements.txt` again after adjusting the file?

**Comment by modrzew** [comment]
Similar changes are already in `forts` branch which I'll be merging to master later today. Thanks for the PR, I'll leave it open until changes land in master.

**Comment by Aiyubi** [comment]
A note: you need the libencrypt fitting to your OS. If you have the wrong one the worker.log should display an error like this one 
`OSError: ./libencrypt.so: wrong ELF class: ELFCLASS64`

**Comment by modrzew** [comment]
Actually @Noctem this is almost identical to what I wrote in `forts`, so I think I'll merge it to master and deal with merge conflicts later on my own.

Could you just change `ENCRYPT` to `ENCRYPT_PATH`?

**Comment by Glennvd** [comment]
Where do you get the libencrypt.so from?

**Comment by Glennvd** [comment]
Here :-)
```[2016-08-07 07:51:21,381][ worker-69][   ERROR][L 132] A wild exception appeared!
Traceback (most recent call last):
  File "worker.py", line 127, in run
    self.main()
  File "worker.py", line 169, in main
    map_objects = response_dict['responses'].get('GET_MAP_OBJECTS', {})```

**Comment by Aiyubi** [comment]
@Glennvd @Eskof it would help if you would not cut the exception out. Also please use code blocks ( 3 times backtick and not just 1)

@Eskof are you using more workers then accounts?

**Comment by Eskof** [comment]
@Aiyubi  No, enough. But I edited, running pip -r install fixed the issue

**Comment by Kieken72** [comment]
Can we convert the .so file to be used on windows?




On Sun, Aug 7, 2016 at 1:49 PM +0200, "Glenn Vandamme" <notifications@github.com<mailto:notifications@github.com>> wrote:


Where do you get the libencrypt.so from?

-

**Comment by Aiyubi** [comment]
@Kieken72 use the dll files I linked in one of my last posts

**Comment by Eskof** [comment]
@Glennvd @Aiyubi I got the same issue as you @Aiyubi  on my other pokeminer I just tried to update, I let you know if I fix it. Did you fixed it?

**Comment by Aiyubi** [comment]
@Glennvd @Eskof this always happens when the login fails. There is an unhandled exception somwhere in the code. It is an old one and not connected to the api change. (I got that one myself and workers will restart after some time)

P.S.: tripple backtick for linebreaks....

**Comment by thiemok** [comment]
Got the same exception as @Eskof posted half an hour ago. However only on x64, armhf runs fine

**Comment by Eskof** [comment]
@thiemok runnin `pip install -r requirements.txt` fixed the issue
@Aiyubi I'm using exactly same setup at both, only worker and location change

**Comment by thiemok** [comment]
@Eskof mb, it was an api bug, they fixed it, but i had the commit pinned in my requirements.txt

**Comment by Aiyubi** [comment]
@Eskof just google "ptc account creator github"

**Comment by Glennvd** [comment]
Got it up and running after recreating my database again, was giving me segmentation faults for some reason 👍

**Comment by Eskof** [comment]
@Aiyubi Thanks I checked it. I dont know what I'm doing wrong, Its stay stuck at "Waiting for email" and fail

**Comment by thiemok** [comment]
@Eskof yeah that tool is currently not working. They did some changes there

**Comment by Eskof** [comment]
@thiemok Any other tools to make some? :(

**Comment by thiemok** [comment]
@Eskof Lets not discuss this here, you might to look trough /r/pokemongodev a bit and brows the issues of that currently not working github project. If you really need to drop me a dm on /u/_chewbaccat

File: requirements.txt (hunk 1 above)
Comment (Brewfasa) [inline requirements.txt]: I get this error when i try to install the new API:
https://i.gyazo.com/4d2a80e943d1ee6f12823dfb2ca5f448.png

Comment (Aiyubi) [inline requirements.txt]: @Brewfasa first this was not the right place to comment. Second: try these things: http://www.pandastack.com/unable-to-find-vcvarsall-bat-how-to-fix-on-windows/

**Comment by modrzew** [comment]
Merged different commit which does exactly the same thing into master, and released that as v0.4.3. Thanks again!

File: requirements.txt (hunk 1 above)
Comment (Brewfasa) [inline requirements.txt]: I know, but it was somehow still kinda connected to the github ^^
I got the compiling working, but now my workers are stuck @ "Exception" status and the logs tell me this:
https://i.gyazo.com/54b34c702cfb58dd9e59abef340d78cd.png

No clue what to do now, i will probably not be the only one getting all these things

Comment (Brewfasa) [inline requirements.txt]: I know, but it was somehow still kinda connected to the github ^^
I got the compiling working, but now my workers are stuck @ "Exception" status and the logs tell me this:
https://i.gyazo.com/54b34c702cfb58dd9e59abef340d78cd.png

No clue what to do now, i will probably not be the only one getting all these things

Comment (Aiyubi) [inline requirements.txt]: @Brewfast with wrong place I mean you are commenting on a line of code and not the issue itself...

Your problem is answered in the other comments. With windows you will need the right dll file and not the so

**Comment by photex12** [comment]
@brewfase
https://i.gyazo.com/4d2a80e943d1ee6f12823dfb2ca5f448.png

I got this problem solved by installing Microsoft Visual C++ Compiler for Python 2.7

**Pull request closed without merging**